students/eowyn/session09/oo-mailroom/test_oomailroom.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+import oomailroom as mr
+import os.path
+
+
+def test_donor_init():
+    name = "Ada Lovelace"
+    amt = 345
+    d = mr.Donor(name)
+    d.add_donation(amt)
+    assert d.name == name
+    assert d.donations == [345]
```

Comment:
the donations list is an implementation detail.

maybe test if most_recent_donation is right, or???